migrate.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+json_data=open('./migrationData.json').read()
```

Comment:
`json_data=open('./migrationData.json', encoding='utf8').read()` (otherwise this will try to read the data as whatever the local terminal's codepage/charmapping is set to. In windows, that means it tries to read it as ANSI)